src/reducers/liquidity_by_token_pair/README.md

```diff
@@ -24,20 +24,23 @@ The reducer was implemented to be used for Redis. Hence, it produces key/value p
 
 A Redis key for a token pair for which at least one liquidity pool exists follows the following schema:
 
-(`<pool_prefix>.`)?(`<currency_symbol_a>.<a_token_name>:` | `<empty>`)(`<currency_symbol_b>.<b_token_name>`)
+(`<pool_prefix>.`)?(`_<currency_symbol_a>.<a_token_name>_:` | `_._`)(`_<currency_symbol_b>.<b_token_name>_`)
 
```

Comment:
there is no longer need for the `?` right? Since the key has now one unique strict format with two tokens

Reply:
The pool prefix can be omitted via the configuration of the reducer. So it is optional.